### DISCUSSION TITLE
Fooocus and Openpose / controlnet

### DISCUSSION THREAD

**Discussion by FrietbakkerThjeu** [lllyasviel/Fooocus#1324] [Q&A] [answered]
Hey there,

is there a way to get poses on fooocus? I'm looking for an openpose integration. How do you guys do poses?

Thanks in advance, great work i really love fooocus

**Comment by IPv6** [comment]
poses are bad on SDXL, no good solutions afaik... SD1.5 openpose controlnets are the only reliable //

**Reply by FrietbakkerThjeu** [reply]
Super detailed prompts i guess

**Comment by ivanryzk** [comment, marked as answer]
This might help: https://github.com/fenneishi/Fooocus-ControlNet-SDXL#Pose

**Reply by FrietbakkerThjeu** [reply]
This is exactly what i was looking for, thanks!

**Reply by IamTirion** [reply]
But the problem is that it is in a different fork, one that is far behind the official one in the core capabilities. It has many problems that were already fixed, and is missing a lot of features that were added from the time it was split from the main branch.

Is there a way to merge this feature into the official Fooocus version manually?

**Reply by IPv6** [reply]
@lllyasviel sorry for tagging - are there any chance for porting pose && depth capabilities of Fooocus-ControlNet-SDXL to Fooocus? They are easy-to-use and somewhat standard now and open many capabilities.

BTW, out of curiosity - why openpose CNs so much better in SD1.5? SDXL seems to be similar in structure (except resolution tagging), but the difference is staggering. Would it be possible to add CNs on refinder stage with SD1.5 model if direct SDXL mangling not archievable?

**Reply by Veemefilms** [reply]
I am very interested in this topic. I am looking to transfer pose to the original fooocus since it is more updated. If you know anything about how I can add CN or Pose to the original fooocus, it will be very grateful. Sorry for my bad English.

**Reply by mirek190** [reply]
Me to.!
To fooocus should be added more controlnet modules.
